[PATCH] From 15274a197e4d3b7ac0f11ca63a5274f1168739f5 Mon Sep 17

From: Pavel Shirshov <pavelsh@microsoft.com>
Date: Mon, 16 Nov 2020 18:11:47 -0800
 00:00:00 2001 Subject: [PATCH 2/8] Reduce severity of 'Vty connected from'
 message
---
 lib/vty.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/vty.c b/lib/vty.c
index 43ff964d8..fad2ff991 100644
--- a/lib/vty.c
+++ b/lib/vty.c
@@ -1959,7 +1959,7 @@ static void vty_accept(struct event *thread)
 		zlog_info("can't set sockopt to vty_sock : %s",
 			  safe_strerror(errno));
 
-	zlog_info("Vty connection from %pSU", &su);
+	zlog_debug("Vty connection from %pSU", &su);
 
 	vty_create(vty_sock, &su);
 }
